lib/utils/btc_address.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:breez/bloc/user_profile/currency.dart';
+import 'package:fixnum/fixnum.dart';
+
+BTCAddressInfo parseBTCAddress(String scannedString) {
+  Uri uri = Uri.parse(scannedString);
+  String address = uri.path;
+  Int64 satAmount;
+  if (uri.queryParameters["amount"] != null)
+    satAmount =
+        Currency.BTC.toSats(double.parse(uri.queryParameters["amount"]));
```

Comment:
We need to be more robust here. `tryParse`?

Reply:
[Done](https://github.com/breez/breezmobile/pull/348/commits/109375e373044e758405ad7f6da0c93af370a922)